### PR TITLE
Feat UI reborn

### PR DESCRIPTION
**Pull request by danni-cool**
# Note: 
I've added a function both on `presets.go` and `detailing.go`, in order to set their own div wrap class, because I found some style differences in Kakuyasu, but not suitable for Qor5.

### CODE REVIEW

**Comment by danni-cool** [comment]
it seems the test builder didn't throw any useful information.

**Review by sunfmin — APPROVED** [review]

**Review by sunfmin — CHANGES_REQUESTED** [review]
Test have to pass

File: presets/presets.go [hunk 1]
```diff
@@ -113,6 +114,11 @@ func New() *Builder {
 	return r
 }
 
+func (b *Builder) SetContainerClass(name string) {
```
Comment (sunfmin) [inline presets/presets.go:117]: 一般我们都不加Set这个词来设置参数，直接用ContainerClass

Comment (sunfmin) [reply presets/presets.go:117]: 如果同一个东西需要Get，那用Get这个词，但是Set这个词是省略的

**Review by sunfmin — CHANGES_REQUESTED** [review]

File: presets/presets.go (hunk 1 above)
Comment (danni-cool) [reply presets/presets.go]: 已经根据规范修改，rebase了master有太多非自己修改，重新提pr

**Pull request closed without merging**